### DISCUSSION TITLE
Shift + Delete 删除自造词后，过段时间又再次出现？

### DISCUSSION THREAD

**Discussion by pdog18** [rime/home#1215] [Q&A]
例如我曾经有一个自造词「shihu = 石斛」 ，因为在我想要输入「shih = 时候」时，这个自造词变成了首选词，所以我希望他不要出现首选，就通过「Shift + Delete」进行了删除，但是好几次发现过了一段时间后(甚至是第二天电脑重新开机后)，这个自造词又出现了，就特别奇怪。

另外请问一下应该用什么办法让「you」总是出现「有」，而不是每次输入一次「又」之后，总是得到「you => 又」。然后总是陷入「删除(又)，下次输入(又)，再次删除(又)」的无奈循环中。

**Comment by DonCN** [comment]
我windows下Shift + Del无法删除候选词。

**Reply by pdog18** [reply]
> 我windows下Shift + Del无法删除候选词。

你直接这样说，我也不知道为什么，这样的发言太没有质量了，如果你想说的是，在默认配置下无法删除候选词，那可能还有点价值(会不会和其他软件热键冲突了，甚至是键盘的问题(笑))。

但是如果你是使用了其他人的配置，然后说无法删除，那谁知道该如何回答你呢?

**Reply by pdog18** [reply]
还有情况也许是，你想删除的「候选词」并不是自造词，是词库中的词，或者是固定码表中(自定义短语)的次，所以不可以删除

**Comment by pdog18** [comment]
> 另外请问一下应该用什么办法让「you」总是出现「有」，而不是每次输入一次「又」之后，总是得到「you => 又」。然后总是陷入「删除(又)，下次输入(又)，再次删除(又)」的无奈循环中。

这个问题的解决，可以使用自定义词库来处理，参见 `custom_phrase`
